api/v1alpha2/awscluster_conversion.go

```diff
@@ -127,6 +127,9 @@ func Convert_v1alpha2_AWSClusterSpec_To_v1alpha3_AWSClusterSpec(in *AWSClusterSp
 	// Manually convert Bastion.
 	out.Bastion.Enabled = !in.DisableBastionHost
 
+	// Manually convert SSHKeyName
+	out.SSHKeyName = &in.SSHKeyName
+
```

Comment:
Do we need to handle the reverse conversion as well?

It might be good to add a test to verify that the down conversion works as expected.